modules/fedimint-mint-client/src/backup/recovery.rs

```diff
@@ -278,12 +284,10 @@ impl From<CompressedBlindedMessage> for BlindedMessage {
 
 #[derive(Debug, Clone, Decodable, Encodable)]
 pub enum MintRecoveryState {
-    V1(MintRecoveryStateV0),
+    #[encodable(index = 2)]
```

Comment:
skipped 1 to keep index in sync with V2, same MintRecoveryStateV2

Reply:
LOL, I had already forgotten I implemented this :see_no_evil:

Reply:
This should have be been a comment. @maan2003

Reply:
this or https://github.com/fedimint/fedimint/pull/6458#discussion_r1858171805

Reply:
@dpc https://github.com/fedimint/fedimint/pull/6464

Reply:
#6464 is good. Without it no one is going to know why this was done.